circle.yml

```diff
@@ -0,0 +1,43 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  cache_directories:
+    - "~/docker"
+  override:
+    - docker info
+    - I="image-$(date +%j).tar";if [[ -e ~/docker/$I ]]; then echo "Loading $I";docker load -i ~/docker/$I;fi
```

Comment:
So this attempts to load an existing `image-<day-of-year>.tar` if one already exists ...